pkgs/os-specific/linux/sgx/sdk/default.nix

```
@@ -46,6 +46,11 @@ stdenv.mkDerivation rec {
   '';
 
   patches = [
+    # Fix missing pthread_compat.h
+    (fetchpatch {
+      url = "https://github.com/intel/linux-sgx/commit/254b58f922a6bd49c308a4f47f05f525305bd760.patch";
+      sha256 = "sha256-sHU++K7NJ+PdITx3y0PwstA9MVh10rj2vrLn01N9F4w=";
+    })
```

I tend to find it clearer to reference the fork of an open PR. Otherwise, the `url` looks more official than it actually is:

```suggestion
    # Fix missing pthread_compat.h: https://github.com/intel/linux-sgx/pull/784
    (fetchpatch {
      url = "https://github.com/jonringer/linux-sgx/commit/254b58f922a6bd49c308a4f47f05f525305bd760.patch";
      sha256 = "sha256-sHU++K7NJ+PdITx3y0PwstA9MVh10rj2vrLn01N9F4w=";
    })
```

---

I'm planning on deleting the fork once it's merged. so I would like to keep it as the upstream.

I can add the PR comment though

---

Alright 👍🏻

---

added